dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04305UpdateWorkflowActionTable.java

```diff
@@ -0,0 +1,596 @@
+    private static final String ORACLE_FIND_REQUIRES_CHECKOUT_OPTION_COLUMN    = "SELECT requires_checkout_option FROM workflow_action";
+
+    private static final String MYSQL_FIND_SCHEME_ID_COLUMN    = "SELECT scheme_id FROM workflow_action";
+    private static final String POSTGRES_FIND_SCHEME_ID_COLUMN = "SELECT scheme_id FROM workflow_action";
+    private static final String MSSQL_FIND_SCHEME_ID_COLUMN    = "SELECT scheme_id FROM workflow_action";
+    private static final String ORACLE_FIND_SCHEME_ID_COLUMN   = "SELECT scheme_id FROM workflow_action";
+
+    private static final String MYSQL_ADD_SCHEME_ID_COLUMN    = "ALTER TABLE workflow_action ADD scheme_id VARCHAR(36) NOT NULL";
+    private static final String POSTGRES_ADD_SCHEME_ID_COLUMN = MYSQL_ADD_SCHEME_ID_COLUMN;
+    private static final String MSSQL_ADD_SCHEME_ID_COLUMN    = "ALTER TABLE workflow_action ADD scheme_id NVARCHAR(36) NOT NULL";
+    private static final String ORACLE_ADD_SCHEME_ID_COLUMN   = "ALTER TABLE workflow_action ADD scheme_id varchar2(36) NOT NULL";
+
+    private static final String MYSQL_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN    = "ALTER TABLE workflow_action ADD requires_checkout_option VARCHAR(16)  default 'both'";
+    private static final String POSTGRES_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN = "ALTER TABLE workflow_action ADD requires_checkout_option VARCHAR(16)  default 'both'";
+    private static final String MSSQL_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN    = "ALTER TABLE workflow_action ADD requires_checkout_option NVARCHAR(16) default 'both'";
+    private static final String ORACLE_ADD_REQUIRES_CHECKOUT_OPTION_COLUMN   = "ALTER TABLE workflow_action ADD requires_checkout_option varchar2(16) default 'both'";
+
+
+    private static final String MYSQL_SELECT_ACTIONS_AND_STEPS = "SELECT wa.id action_id, ws.id step_id, wa.my_order FROM workflow_step ws INNER JOIN workflow_action wa ON ws.id = wa.step_id";
+    private static final String POSTGRES_SELECT_ACTIONS_AND_STEPS = MYSQL_SELECT_ACTIONS_AND_STEPS;
+    private static final String MSSQL_SELECT_ACTIONS_AND_STEPS = MYSQL_SELECT_ACTIONS_AND_STEPS;
+    private static final String ORACLE_SELECT_ACTIONS_AND_STEPS = MYSQL_SELECT_ACTIONS_AND_STEPS;
+
+    private static final String MYSQL_INSERT_INTO_INTERMEDIATE_TABLE    = "INSERT INTO workflow_action_step (action_id, step_id, action_order) VALUES (?, ?, ?)";
+    private static final String POSTGRES_INSERT_INTO_INTERMEDIATE_TABLE = MYSQL_INSERT_INTO_INTERMEDIATE_TABLE;
+    private static final String MSSQL_INSERT_INTO_INTERMEDIATE_TABLE    = MYSQL_INSERT_INTO_INTERMEDIATE_TABLE;
+    private static final String ORACLE_INSERT_INTO_INTERMEDIATE_TABLE   = MYSQL_INSERT_INTO_INTERMEDIATE_TABLE;
+
+    /*private static final String MYSQL_SELECT_SCHEME_IDS_FOR_ACTIONS = "SELECT wa.id action_id, ws.scheme_id, wa.requires_checkout FROM workflow_action wa INNER JOIN workflow_step ws ON ws.id = wa.step_id";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 207 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)